src/preact/base-element.js

```diff
@@ -190,34 +191,46 @@ const IS_EMPTY_TEXT_NODE = (node) =>
  * @template API_TYPE
  */
 export class PreactBaseElement extends AMP.BaseElement {
-  /**
-   * @return {boolean}
-   * @nocollapse
-   */
+  /** @override @nocollapse */
+  static V1() {
+    return true;
+  }
+
+  /** @override @nocollapse */
   static requiresShadowDom() {
     // eslint-disable-next-line local/no-static-this
     return this['usesShadowDom'];
   }
 
+  /** @override @nocollapse */
+  static prerenderAllowed() {
+    // eslint-disable-next-line local/no-static-this
+    const Ctor = this;
+    return !Ctor['loadable'];
```

Comment:
Shouldn't this be `!!`?
```suggestion
    return !!Ctor['loadable'];
```

Reply:
prerender should only be allowed if it _isn't_ loadable. aka it doesn't make any network requests.

Reply:
Correct. Only `!loadable` are prerenderable by default. Also, as mentioned in [this thread](https://github.com/ampproject/amphtml/pull/32974#discussion_r588830001), I thought it was a good default. But LMK if you disagree and this is too risky.